app/models/task.py

```diff
@@ -1,5 +1,25 @@
 from app import db
 
-
 class Task(db.Model):
-    task_id = db.Column(db.Integer, primary_key=True)
+    task_id = db.Column(db.Integer, primary_key=True, autoincrement=True)
+    title = db.Column(db.String)
+    description = db.Column(db.String)
+    completed_at = db.Column(db.DateTime, nullable=True, default = None)
+    goal_id = db.Column(db.Integer, db.ForeignKey("goal.goal_id"), nullable=True)
+    def to_dict(self):
+        if Task.completed_at is True:
+            task_as_dict["is_completed"] == True
+        else:  
+            task_as_dict = {}
+            task_as_dict["id"] = self.task_id
+            task_as_dict["title"] = self.title
+            task_as_dict["description"] = self.description
+            task_as_dict["is_complete"] = (self.completed_at != None)
```

Comment:
This line reads well to me. It says "The dictionary `task_as_dict` should have a key-value pair, where the key is `"is_complete"`. The value is equal to `self.completed_at != None`, which will either be `True` or `False`."

For code style, we could remove the parentheses `()`:
```python
task_as_dict["is_complete"] = (self.completed_at != None)
```